web/src/features/hfiCalculator/components/HFISuccessAlert.tsx

```diff
@@ -0,0 +1,45 @@
+import { makeStyles } from '@material-ui/core/styles'
+import { Alert } from '@material-ui/lab'
+import { Snackbar } from '@material-ui/core'
+import React from 'react'
+
+export interface HFISuccessAlertProps {
+  message: string | null
+}
+
+const useStyles = makeStyles({
+  alert: {
+    backgroundColor: '#2E8540',
+    color: 'white',
+    '& .MuiAlert-icon': {
+      color: 'white'
+    },
+    marginTop: '120px'
```

Comment:
Minor: we usually just set pixel properties with a number, not a string, e.g.
`marginTop: 120`